cluster-autoscaler/cloudprovider/externalgrpc/README.md

```diff
@@ -56,8 +56,8 @@ To regenerate the gRPC code:
 1. install `protoc` and `protoc-gen-go-grpc`:
 
 ```bash
-go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.26
-go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1
+go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
```

Comment:
Updated to latest version at the time of creating this PR. 
![image](https://github.com/kubernetes/autoscaler/assets/34534103/9f149db1-3afe-4d7a-984f-faffa9047861)
https://grpc.io/docs/languages/go/quickstart/